main.go

```diff
@@ -139,7 +140,10 @@ func Main() {
 	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
 	<-sigs
 
-	logger.Log().Msg("JetKVM Shutting Down")
+	logger.Info().Msg("JetKVM Shutting Down")
```

Comment:
Please swap with the `stopAudio()` call (do that first)

Also, we used `logger.Log()` here because we _always_ want to log out this message.